package/com.unity.formats.usd/TestRunnerOptions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "disableBatchMode": true
```

Comment:
Just for my own understanding, why was this needed?
It sucks that we have no way to document this in the json :/

Reply:
Tests that rely on the rendering need a valid gameView open (to generate renders).
By default upm-ci runs runtime tests in batch mode (no rendering) causing tests to hang or straight up not render. This runs unity with a HEAD :) (probably a tad slower than batch mode).